src/frontend/Komponenter/Behandling/Brevmottakere/SøkWrapper.tsx

```diff
@@ -17,10 +16,6 @@ enum ESøktype {
     PERSON = 'PERSON',
 }
 
-const StyledIngress = styled(Ingress)`
-    margin-bottom: 1rem;
-`;
-
 const SøkTypeSelect = styled(Select)`
```

Comment:
```suggestion
const DropDown = styled(Select)`
```
evt. `DropDownMenu` (blir kanskje litt langt)
evt. `Nedtrekksmeny`